resources/scripts/jquery/jquery-utils.js

```diff
@@ -304,14 +304,40 @@
     		$(pages[i]).css("display", "none");
     	}
     	$(".eXist_wizard_back", container).button("disable");
-    	
+    	$(".eXist_wizard_done", container).button("disable");
     	$(".eXist_wizard_next", container).click(function () {
     		if (currentPage == pages.length - 1)
     			return;
+		const name = document.getElementsByName('name')[0].value;
+           	const title = document.getElementsByName('title')[0].value;
+
+                if (name.startsWith('file:')) {
+                    alert("The name cannot use a file:// protocol.")
+                    return false;} else {
+                       try {
+                           new URL(name);
+                           } catch (_) {
+                           return alert("Enter a valid URI as name.");
+                        }
+                    }
+		
+		const abbrev = document.getElementsByName('abbrev')[0].value;
+                
+		if (abbrev == "") {
+                alert("Enter a valid abbreviation");
+                return false;
+           	};
+		
+                if (title == "") {
```

Comment:
Since you agreed I would then also move the title declaration here.

Reply:
Done